admin/data/sql/migrations/20190918140901.do.check-state.sql

```diff
@@ -0,0 +1,21 @@
+alter table mtc_admin.[check]
+	add received bit not null
+  constraint DF_received_Default default 0
+
+alter table mtc_admin.[check]
+	add complete bit not null
+  constraint DF_complete_Default default 0
+
+alter table mtc_admin.[check]
+	add completedAt datetimeoffset(3)
+
+alter table mtc_admin.[check]
+	add processingFailed bit not null
+  constraint DF_processingFailed_Default default 0
```

Comment:
If processing has failed do we record why it failed? E.g. failed validation field X missing.  This is not for teachers but for support/classification and reporting.

Reply:
yes, that is in receivedCheck (table storage).